### PR TITLE
Add support for Modernizr in docs and raw

### PR DESCRIPTION
**Pull request by Scotchester**
Also tweaks change log format slightly to account for new recommendations
by keepachangelog.com.

And some ancient typo fix.

### CODE REVIEW

File: docs/index.jade [hunk 1]
```diff
@@ -21,10 +21,8 @@ html(lang="en", class="no-js")
         link(rel = 'stylesheet', type = 'text/css', href = 'static/docs/brackets.css')
         link(rel = 'stylesheet', type = 'text/css', href = 'static/docs/docs.css')
         link(rel = 'stylesheet', type = 'text/css', href = document.relativeSource)
-        script.
-            // Confirm availability of JS and remove no-js class from html
-            var docElement = document.documentElement;
-            docElement.className = docElement.className.replace(/(^|\s)no-js(\s|$)/, '$1$2');
+        if(templateData.modernizr)
+            script(src="static/js/modernizr-custom.js")
```
Comment (Scotchester) [inline docs/index.jade]: It does, in fact, replace `.no-js`!

Comment (Scotchester) [inline docs/index.jade]: I did, though, change the Modernizr task config to not use the `moz-` prefix, so we don't have to update any uses of `.js` to `.mod-js`.

Comment (himedlooff) [inline docs/index.jade]: Ah, thanks!

**Comment by himedlooff** [comment]
Remember to tag and release.

**Pull request merged**